Composer/packages/webchat-client/src/utils/ConversationService.ts

```diff
@@ -0,0 +1,316 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { v4 as uuidv4 } from 'uuid';
+import axios, { AxiosInstance } from 'axios';
+import { createDirectLine } from 'botframework-webchat';
+
+export const headers = {
+  'Content-Accept': 'application/json',
+};
+
+export type User = {
+  id: string;
+  name: string;
+  role: string;
+};
+
+export type BotSecrets = { msAppId: string; msPassword: string };
+export type ChannelService = 'public' | 'usgov';
+
+type WebChatMode = 'livechat' | 'transcript';
+type ConversationMember = {
+  id: string;
+  name: string;
+  role?: string;
+};
+
+type StartConversationPayload = {
+  bot?: ConversationMember;
+  botUrl: string;
+  channelServiceType: ChannelService;
+  members: ConversationMember[];
+  mode: WebChatMode;
+  locale: string;
+  msaAppId?: string;
+  msaPassword?: string;
+};
+
+// All activity types. Just parsing for Trace currently.
+export enum ActivityTypes {
```

Comment:
nit: ActivityType (singular)
for reference:
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations